spec/features/users_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.feature 'Users', type: :feature do
+  context 'registrations users' do
```

Comment:
Probably you wanted to write all tests related to users in this file 🤔
Split them if that is the reason for this context. So, you will have `user_login_spec.rb`, `user_registration_spec.rb`, `user_show_spec.rb`. Then you won't have a very long file with all specs related to users and it will be easier to find something for other devs.